### DISCUSSION TITLE
How do I get Player object of player, who ran command, inside command.

### DISCUSSION THREAD

**Discussion by AntonIXO** [Minestom/Minestom#2111] [Help] [answered]
I want to write command to save instance of player. 
To do this I need to get instanceContainer of player inside command:
`
Player player = MinecraftServer.getCommandManager().getConsoleSender().asPlayer();
Instance instance = player.getInstance();
instance.saveChunksToStorage();
`
This one is not working for me and .asPlayer is deprecated.
How can I implement this better?

**Comment by vearendev** [comment, marked as answer]
```
public final class ExampleCommand extends Command {
   public ExampleCommand() {
       super("test");
       addSyntax(this::onTestCommand);
   }

   private void onTestCommand(CommandSender sender, CommandContext context) {
       if (!(sender instanceof Player player)) {
           return;
       }
       final Instance playerInstance = player.getInstance();
   }
}
```
